src/main/java/com/launchdarkly/client/FeatureFlagBuilder.java

```diff
@@ -0,0 +1,83 @@
+package com.launchdarkly.client;
+
+import com.google.gson.JsonElement;
+
+import java.util.List;
+
+public class FeatureFlagBuilder {
+  private String key;
+  private int version;
+  private boolean on;
+  private List<Prerequisite> prerequisites;
+  private String salt;
+  private String sel;
+  private List<Target> targets;
+  private List<Rule> rules;
+  private Rule fallthrough;
+  private Integer offVariation;
+  private List<JsonElement> variations;
+  private boolean deleted;
+
+  public FeatureFlagBuilder(String key) {
```

Comment:
Let's add a ctor for `FeatureFlagBuilder(FeatureFlag f)` and then make `FeatureFlag` immutable.